server/database.js

```diff
@@ -30,6 +30,7 @@ class Database {
     static patchList = {
         "patch-setting-value-type.sql": true,
         "patch-improve-performance.sql": true,
+        "patch-add-retry-interval-monitor.sql": true
```

Comment:
```suggestion
        "patch-add-retry-interval-monitor.sql": true,
```

Reply:
I think I'm missing something here? But a `,` is not needed on the last key/value pair of a object.

Reply:
We have `comma-dangle` enabled in ESLint (https://eslint.org/docs/rules/comma-dangle)

Reply:
Ahh, I see now. My VSCode did not tell me that was a problem (ESLint extension is installed). Hmm strange 🤔

Do I make the changes locally, or press the "Commit suggestions" to use your suggested changes?
(New to contributing on projects on GitHub 😅)

Reply:
Use commit suggestions button. This is faster than editing file, commiting and pushing ;)

Reply:
Thank you for the help ☺